src/pipecat/services/tavus/test_scripts/create_persona.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+API_URL="https://tavusapi.com/v2/personas"
+
+curl --request POST \
+  --url "$API_URL" \
+  --header "Content-Type: application/json" \
+  --header "x-api-key: $TAVUS_API_KEY" \
+  --data '{
+    "persona_name": "Pipecat",
+    "system_prompt": "",
+    "pipeline_mode": "echo",
+    "context": "",
+    "layers": {
+      "transport":{
+        "input_settings":{
+            "microphone": {
+                "isEnabled": true
+            }
```

Comment:
It is currently only supported sending `app-messages` to transmit audio. I should remove this script before merging this PR.